workers/generic-worker/main.go

```diff
@@ -207,6 +207,10 @@ func loadConfig(configFile *gwconfig.File) error {
 			DownloadsDir:                   "downloads",
 			IdleTimeoutSecs:                0,
 			LiveLogExecutable:              "livelog",
+			// The ports on which the livelog process listens locally.  These ports are not exposed
+			// outside of the host.  However, in CI they must differ from those of the generic-worker
+			// instance running the test suite.
```

Comment:
The "in CI" in this comment means "in the g-w tests", so I think this comment would be better where the value is set to 30538 in `helper_test.go`.